### PR TITLE
[IMP] add license file for the mail addins reposiroty

### PR DESCRIPTION
**Pull request by std-odoo**

### CODE REVIEW

File: COPYRIGHT [hunk 1]
```diff
@@ -0,0 +1,15 @@
+
+Most of the files are
+
+  Copyright (c) 2004-2015 Odoo S.A.
```
Comment (tde-banana-odoo) [inline COPYRIGHT:4]: 2015 :D ?

Comment (std-odoo) [reply COPYRIGHT:4]: I just copy / paste from the odoo repo, not sure if I can change it or not :/

Comment (tde-banana-odoo) [reply COPYRIGHT:4]: I suppose we can say "TODAY" without that much impact.

Comment (std-odoo) [reply COPYRIGHT:4]: Indeed, looks like the date is the "claim date", (the plugin didn't exists in 2014) https://stackoverflow.com/questions/2390230/do-copyright-dates-need-to-be-updated

So I update it :)

**Review by grindtildeath — APPROVED** [review]
Thanks, waiting for the merge :smiley:

File: COPYRIGHT (hunk 1 above)
Comment (tde-banana-odoo) [reply COPYRIGHT]: Shouldn't it start on 2020 or 2021 ?

Comment (std-odoo) [reply COPYRIGHT]: IDK if we can put a year in the past...

Comment (std-odoo) [reply COPYRIGHT]: ok, thanks for the explanation :)

**Comment by grindtildeath** [comment]
ping @std-odoo @tde-banana-odoo @odony 

Can you guys please merge the PR? :pray:

**Comment by tde-banana-odoo** [comment]
Done :)

**Pull request merged**